volatility3/cli/volshell/windows.py

```diff
@@ -44,11 +44,58 @@ def list_processes(self):
             )
         )
 
+    def get_process(self, pid=None, v_offset=None, p_offset=None):
+        """Returns the EPROCESS object that matches the pid. If v_offset/p_offset is provided, construct the EPROCESS object at the provided address. Only one parameter is allowed."""
```

Comment:
Same here with regards to `Args:` and `Returns:`.  Also I kinda prefer variables have more descriptive names (although I'll allow it if you really want, since it is likely to be typed be developers quite often).